examples/adminapi.py

```diff
@@ -419,17 +420,92 @@ def example_list(a, args):
         print(" {} consumer groups".format(len(groups)))
         for g in groups:
             if g.error is not None:
-                errstr = ": {}".format(t.error)
+                errstr = ": {}".format(g.error)
             else:
                 errstr = ""
 
             print(" \"{}\" with {} member(s), protocol: {}, protocol_type: {}{}".format(
-                  g, len(g.members), g.protocol, g.protocol_type, errstr))
+                g, len(g.members), g.protocol, g.protocol_type, errstr))
 
             for m in g.members:
                 print("id {} client_id: {} client_host: {}".format(m.id, m.client_id, m.client_host))
 
 
+def example_delete_consumer_groups(a, args):
+    groups = a.delete_consumer_groups(args, timeout=10)
+    for group_id, future in groups.items():
+        try:
+            response = future.result()
```

Comment:
don't need to use `result`. maybe `future.result()  # empty, but raises exception on failure` like other example above.